### PR TITLE
Show build status message when it exists

### PR DESCRIPTION
**Pull request by jwforres**
Implements https://trello.com/c/Of6LXNUi

### CODE REVIEW

**Comment by jwforres** [comment]
status-message

**Comment by jwforres** [comment]
@spadgett 

Note there are currently some issues with the messages showing up so i could only test cancelled https://github.com/openshift/origin/issues/12146

**Comment by spadgett** [comment]
Should we add it to the build config page as well for latest build?

https://github.com/spadgett/origin-web-console/blob/a6e9dcb8aa111a0ccbbb92f8e1036a2d7c8b3341/app/views/browse/build-config.html#L136-L163

**Comment by jwforres** [comment]
@spadgett updated

File: app/views/browse/build-config.html [hunk 1]
```diff
@@ -197,13 +198,19 @@
                               <span ng-if="!(build | annotation : 'buildNumber')">
                                 <a ng-href="{{build | navigateResourceURL}}">{{build.metadata.name}}</a>
                               </span>
-                              <span ng-if="build.status.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-trigger="hover" dynamic-content="{{build.status.message}}"></span>
                             </td>
                             <td data-title="Status">
                               <div row class="status">
-                                <build-status build="build"></build-status>
+                                <!-- <build-status build="build"></build-status> -->
```
Comment (spadgett) [inline app/views/browse/build-config.html]: Commented out code

**Comment by jwforres** [comment]
fixed [merge]

**Pull request merged**